core/domain/email_manager.py

```diff
@@ -77,6 +77,46 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+_EDITOR_ROLE_EMAIL_SUBJECT = (
```

Comment:
For people who want to set up their own Oppia server. The email templates here are Oppia.org-specific.